.github/workflows/oneshotSnapshotPackage.yml

```diff
@@ -0,0 +1,64 @@
+name: Release oneshot snapshot package
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release_snapshot_package:
+    name: "Tag snapshot release"
```

Comment:
What is this doing that is different from releaseSnapshotPackage.yml?

Reply:
same. So we can build a release snapshot on demand.